counterparty-core/counterpartycore/cli.py

```diff
@@ -363,7 +363,7 @@ def float_range_checker(arg):
     [
         ("--electrs-url",),
         {
-            "help": "the URL of the Electrs server",
+            "help": "the complete URL of the Electrs API, for example: `https://mempool.space/api`",
```

Comment:
we should indicate whether you can include the port

Reply:
done.